### PR TITLE
[FIX] l10n_cl: change account in COA and assignation

### PR DESCRIPTION
**Pull request by Danisan**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

This PR replaces https://github.com/odoo/odoo/pull/164077

(odoo-dev:saas-16.3-16.0-fix-account-f29-bAgH-fw)

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_cl/data/l10n_cl_chart_data.xml [hunk 1]
```diff
@@ -115,4 +115,4 @@
         <field name="applicability">accounts</field>
     </record>
 
-</odoo>
+</odoo>
```
Comment (rugo-odoo) [inline addons/l10n_cl/data/l10n_cl_chart_data.xml:118]: We have to keep the blank line here (after `</odoo>`).

File: addons/l10n_cl/data/account_fiscal_template.xml [hunk 2]
```diff
@@ -0,0 +1,134 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+
+        <record id="afpt_non_recoverable_vat_1" model="account.fiscal.position.template">
+            <field name="name">Compras - destinadas a generar operaciones no gravadas o exentas</field>
```
Comment (rugo-odoo) [inline addons/l10n_cl/data/account_fiscal_template.xml:6]: Names should be in english and translated in the `.po` file to the localization language. It is easier to us to maintain (we all speak english).

**Review by rugo-odoo — CHANGES_REQUESTED** [review]
Thanks for your contribution ! Just a couple comments. Also I asked a PO to make a little functional check.

**Comment by fvz-odoo** [comment]
@Danisan hello! could you please address @rugo-odoo's comments?

Let us know, please!

Fernando

File: addons/l10n_cl/data/account_fiscal_template.xml [hunk 3]
```diff
@@ -0,0 +1,134 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
```
Comment (jco-odoo) [inline addons/l10n_cl/data/account_fiscal_template.xml:3]: @Danisan This entire XML has been replaced by csv and templates from saas-16.3.

**Pull request closed without merging**